package.json

```diff
@@ -10,6 +10,9 @@
     "canvas": "yarn extender:build && yarn --cwd project-canvas dev",
     "canvas:build": "yarn extender:build && yarn --cwd project-canvas build ",
     "canvas:test": "yarn --cwd project-canvas test",
+    "eslint": "eslint",
+    "prettier": "prettier .",
+    "typecheck": "tsc --noEmit --jsx react-native   --module esnext --moduleResolution node --resolveJsonModule",
```

Comment:
Please read the last review again.

Reply:
are you pointing to the name of the command ? if so what sould be a meaningfull name for running eslint and prettier tests so that the name of the library is not included in the command ? i suggest "eslint": "eslint", "prettier": "prettier .",

Reply:
that and this "typechecking is tricky to run for everything at once. I suggest you run one for canvas and one for extender"
The way it was done there ignores the tsconfig that was set for each project, and as each project has its own config it's better to run the command from the project itself. So there should be a typecheck command in extender and another one in prettier so that it uses the project own config, and you would not have to override and specify any flag. The ones in root should then just execute those commands.